### ISSUE TITLE
Invalid items in "Magic Storage"

### ISSUE THREAD

**Issue by Tearlow** [srtsnlt/AutoStacker#2]
I know the mod is WIP but figured I'd throw a bug your way.

I see non-items showing up in "Magic Storage" I created this system just to make sure. What causes it I could not tell or what the item is as there are no tooltips for it. I don't really know how to debug this on my own but if you let me know (if at all possible) I can help out.

![2019-02-27_10-09-00](https://user-images.githubusercontent.com/786246/53478964-518a1080-3a78-11e9-9a49-aa0182961e32.png)

**Comment by srtsnlt** [comment]
Thank you for contact me the bug!
I fixed the bug by v0.3.15 that releaced just before..
And, in my environment, normal operation by the update!
Update and try to check operation please!

**Comment by Tearlow** [comment]
Cheers for the update; I'll have a proper look when I get some free time myself. Till then though~ Are there any ways of deleting these non-items from existing cells? Since Magic Storage requires the storage to be empty in order to destroy it...

The only thing I can think of is something like TEdit but would like to avoid that route.

**Comment by Tearlow** [comment]
Right. So I've finally been able to sit down some more with Terraria and this mod and after some intense testing I think it's safe to say this has indeed been solved! (20ish minutes of constant items and not a single invalid item.) I think it is safe to close this.

Tested using 0.3.25
